Spanner/tests/Unit/ResultTest.php

```diff
@@ -253,13 +253,14 @@ public function testSnapshot()
 
     public function testUsesCorrectDefaultFormatOption()
     {
+        $fixture = $this->getStreamingDataFixture()['tests'][1];
         $mapper = $this->prophesize(ValueMapper::class);
         $mapper->decodeValues(
             Argument::any(),
             Argument::any(),
-            'nameValuePair'
-        );
-        $result = $this->getResultClass([], 'r', $mapper->reveal());
+            'associative'
```

Comment:
callout here also - the "default" format option seems to be `associative` and not `nameValuePair`. I think at this point, even if it's the wrong behavior it'd be breaking to change it, but still it's a point of interest.